package.json

```diff
@@ -3,6 +3,7 @@
     "@prisma/client": "2.11.0-dev.25",
     "arg": "4.1.3",
     "fetch-retry": "4.0.1",
+    "graphql": "15.4.0",
```

Comment:
Ugh is this really the right place to do this?

Reply:
Why not?